tests/test_playbooks/search_facts.yml

```diff
@@ -3,10 +3,21 @@
   gather_facts: false
   vars_files:
     - vars/server.yml
+    - vars/search_facts.yml
   tasks:
     - include: tasks/organization.yml
       vars:
+        organization_name: "{{ item.organization }}"
         organization_state: present
+      loop: "{{ test_resources }}"
```

Comment:
I am not sure, i understand the purpose of the `test_reseources` variable, but should we add a `when: item.organization` here?

Reply:
oh the purpose was purely that I don't have to write individual create/destroy calls and not to repeat the same data multiple times.

IMHO item.org will always be set, so no need for the when

Reply:
As the list is rather short, we could just explode it here...

Reply:
I'd prefer to keep it in a vars file. Would a different variable name help in any way?